README.md

```diff
@@ -57,6 +57,16 @@ Git Town is configured on a per-repository basis. It requires two pieces of info
 Upon first use in a repository, you will be prompted for this information.
 Use the [git town](/documentation/commands/git-town.md) command to view or update your configuration at any time.
 
+#### More Configuration
+
+The following are also configured on a per-repository basis.
```

Comment:
The following configuration options have usable defaults, so the configuration wizard does not ask about them. 
Use the [git town](/documentation/commands/git-town.md) command to view or update them for a particular repository if needed.